core/sqf/sqenvcom.sh

```diff
@@ -95,17 +95,20 @@ cpucnt=$(grep processor /proc/cpuinfo | wc -l)
 #     no number means unlimited, and will swamp the system
 export MAKEFLAGS="-j$cpucnt"
 
+# For now set up the TOOLSDIR, it may be overwritten later when the
+# .trafodion configuration file is loaded.
 if [[ -n "$CLUSTERNAME" ]]; then
-  export MY_ROOT=/opt/hp
-  export TOOLSDIR=${TOOLSDIR:-/home/tools}
+  if [ -z "$TOOLSDIR" ]; then
+    export TOOLSDIR=${TOOLSDIR:-/home/tools}
+  fi
```

Comment:
No, this prevents the TOOLSDIR from being overwritten to either /opt/home/tools or /home/tools.  On a VM or a generic system, the tools directory does not exist. If you invoke sqenvcom.sh after setting your own TOOLSDIR envvar and don't create the .trafodion file, your tools directory variable is overwritten and a subsequent build fails.